test_regress/t/t_class2.pl

```diff
@@ -11,13 +11,12 @@
 scenarios(simulator => 1);
 
 compile(
-    fails => $Self->{vlt_all},
-    expect_filename => $Self->{golden_filename},
     );
 
 execute(
     check_finished => 1,
-    ) if !$Self->{vlt_all};
+    expect_filename => $Self->{golden_filename},
```

Comment:
```suggestion
```
For the now-passing tests please remove expect_filename and delete the corresponding .out.  The test should $stop if anything is wrong and otherwise $finish, so there's no need to do a relatively slow I/O comparison.  Thanks